test/data/echopulse/pcschema.xml

```diff
@@ -3,66 +3,66 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
because we still use it in tests  https://github.com/LI3DS/fdw-li3ds/pull/21/files#diff-4c496b3798183f8de6cfdcde6b0293b6R78